opennms-container/horizon/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN groupadd -g 10001 opennms && \
 FROM horizon-base
 
 # A default which installs the required OpenNMS Horizon packages
```

Comment:
Please leave a comment why we add xmp here

Reply:
Agreed. This is going to need to be revisited when merging this back to develop.

Reply:
It is ok for merging for now. I have added comment in PE-161